help/grandorgue.xml

```diff
@@ -6261,6 +6270,13 @@ closed with changed metronome settings.
         </note>
       </sect2>
     </sect1>
+    <sect1 id="update-checking">
+      <title>Updates</title>
+      <sect2>
+        <title>Check for updates at startup</title>
+        <para>Check the checkbox here if you would like GrandOrgue to notify you when a new version of GrandOrgue is available.</para>
```

Comment:
```suggestion
        <para>Check the checkbox here if you would like GrandOrgue to notify you when a new version of GrandOrgue is available. If there is one, GrandOrgue will create a new menu <link linkend="update-available">Update available!</link>.</para>
```
Next, create a description of the Update available! Menu menu with all entries after the description of the Help Menu.